README.md

```diff
@@ -35,6 +35,12 @@ this is entirely optional and depending on your use case, you may use it.
 $ docker network create --subnet=10.0.0.0/24 -d plumgrid -o router=router-1 net1
 ```
 
+You can also choose to link the network to a specific bridge in PLUMgrid virtual domain,
+this also is optional and can be used depending upon your use case.
+```
+$ docker network create --subnet=10.0.0.0/24 -d plumgrid -o bridge=bridge-1 net1
```

Comment:
Bad thread reply, sending on new mail thread.

On Thu, Jul 28, 2016 at 6:59 PM, Eduard Serra <eduards@plumgrid.com> wrote:

> Guys also, I have a question/request. Since I lack knowledge on the cnm
> area I'm blindly asking at this point.
>
> I'm finding incredibly troublesome the fact that we are using uuids for
> the network VNF names that are specified and contained inside a tenant. I
> understand that UUIDs are good to give uniqueness, specially since docker
> has no domain segmentation, so ie. 2 tenants might try to create 2 networks
> that have the same name (which in any case probably docker would reject
> anyway I guess).
>
> IE.
> docker network create -d plumgrid --subnet 10.0.0.0/24 --gateway
> 10.0.0.100 -o domain=<domain_name> <bridge_name>
>
> As read on UI (virtual domain):
> the bridge name:
> bri136b0fbd91c1ac2b92464aba469078b7647de1531473b9139faaa4139b3c9754
>
> cnf: cnf136b0fbd91c1ac2b92464aba469078b7647de1531473b9139faaa4139b3c9754[object]
>
> rule: rules136b0fbd91c1ac2b92464aba469078b7647de1531473b9139faaa4139b3c9754[object]
>
> The current naming convention is hidden on the implementation and is
> pretty much a mix of plumgrid definition and docker definition.
> Is there a chance we can change this to more human-readable,
> *plumgrid-dependable* names?
>
> IE.
> With the example above, I'd expect I to be able to connect *anything* on
> the bridge *without* docker interaction.
> Using the docker uuids is adding an extra dependency for plumgrid to
> configure anything, since now I need first ping docker to check what is the
> associated uuid to that network in order to do anything at all; which from
> a plumgrid POV I don't see so legitimate as *I know perfectly what it is
> and where it is *(domain name and bridge name).
>
> Also consider that, if for any reason, docker looses the data for the
> networks it has created (which stores in the kv store), and the uuids are
> gone, there is little other option other than wiping the network on the
> entire setup anew, since I don't think you can re-create the network that
> are still available in plumgrid on docker with the same uuid it used to
> have.
>
> tl;dr
> Is it possible to find a way we use domain/bridge_name to create a name
> convention that works for us, provides the same uniqueness as the uuids of
> docker have and is still detached from docker uuids? I think it would be
> better for everyone in all possible angles.
>
>
>
>
> On Thu, Jul 28, 2016 at 6:24 PM, Fawad Khaliq <notifications@github.com>
> wrote:
>
>> In README.md
>> <https://github.com/plumgrid/libnetwork-plugin/pull/44#discussion_r72654627>
>> :
>>
>> > @@ -35,6 +35,12 @@ this is entirely optional and depending on your use case, you may use it.
>> >  $ docker network create --subnet=10.0.0.0/24 -d plumgrid -o router=router-1 net1
>> >  ```
>> >
>> > +You can also choose to link the network to a specific bridge in PLUMgrid virtual domain,
>> > +this also is optional and can be used depending upon your use case.
>> > +```
>> > +$ docker network create --subnet=10.0.0.0/24 -d plumgrid -o bridge=bridge-1 net1
>>
>> Let's call it network_id instead of bridge
>>
>> —
>> You are receiving this because you are subscribed to this thread.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/plumgrid/libnetwork-plugin/pull/44/files/3b43366ec70b41e6a6caa7845c030dec623fe5d9#r72654627>,
>> or mute the thread
>> <https://github.com/notifications/unsubscribe-auth/AFP7tFOcVYZaDi2qR_U6QBS8f3Yj4ATiks5qaNe9gaJpZM4JXBkr>
>> .
>>
>
>
>
> --
> Eduard Serra,
> MTS @ Plumgrid Inc
>



-- 
Eduard Serra,
MTS @ Plumgrid Inc